src/khoj/routers/api.py

```diff
@@ -307,8 +308,11 @@ async def extract_references_and_questions(
     for filter in [DateFilter(), WordFilter(), FileFilter()]:
         defiltered_query = filter.defilter(defiltered_query)
     filters_in_query = q.replace(defiltered_query, "").strip()
-
+    conversation = await sync_to_async(ConversationAdapters.get_conversation_by_id)(conversation_id)
+    for filter in conversation.file_filters:
+        filters_in_query += f" file:{filter}"
```

Comment:
This should be `f" file:\"{filter}\"". Currently, it's not actually applying the filter. I tested & fixed it locally, so I'll just push my fix.